File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    answer = 0
    N = len(number)
    for i in range(N-2):
        for j in range(i+1, N-1):
            for k in range(j+1, N):
                tmp = number[i] + number[j] + number[k]
                if tmp == 0:
                    answer += 1
    return answer

File: 프로그래머스/1/131705. 삼총사/삼총사.java
class Solution {
    public int solution(int[] number) {
        int answer = 0;
        int N = number.length;
        
        for (int i = 0; i < N-2; i++) {
            for (int j = i+1; j < N-1; j++) {
                for (int k = j+1; k < N; k++) {
                    int tmp = number[i] + number[j] + number[k];
                    if (tmp == 0) {
                        answer++;
                    }
                }
            }
        }
        return answer;
    }
}